scripts/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Should we be using base/root config files for eslint and tsconfig that the subprojects inherit from (and can override)? I'm assuming in the majority of cases we don't need custom settings per project.